tools/zbdump

```diff
@@ -71,9 +75,13 @@ if not kb.is_valid_channel(args.channel):
 kb.set_channel(args.channel)
 kb.sniffer_on()
 
+if args.channel == 0: rf_freq_mhz = 868
+elif args.channel >= 11 and args.channel <= 26: rf_freq_mhz = ((args.channel - 10) * 5 + 2400)
+elif args.channel >= 1 and args.channel <= 10: rf_freq_mhz = (args.channel * 2 + 904)
+elif args.channel >= 128 and args.channel <= 131: rf_freq_mhz = ((args.channel - 128) * 2 + 780)
```

Comment:
@AdamLaurie I believe you instead were implementing different regions using pages, right? I presume that we should instead have this code follow your behavior, perhaps with adding page CLI options? Or, can we get it all lined up so that users always specify only a channel, and somewhere in the stack converts it to channel/page combinations?

I'm fine with keeping it all channel through the stack -- _but_ implementation needs to match between these changes and other areas of the code. I don't want two different code paths for frequency pages.

Reply:
Could you please be more specific about the 'pages'?
You can see in the documentation (https://www.sewio.net/open-sniffer/develop/http-rest-interface/) that the channel specifies the frequency.

I used a proxy to get verify the requests and every request goes to:
http://10.10.10.2/settings.cgi?chn=XX&modul=XX&rxsens=XX

Reply:
I'll let @AdamLaurie  share his thoughts and then we can figure out how to handle this.

Reply:
Thanks @rmspeers - the page thing is how it's done in Europe and/or on Silicon Labs hardware but it's not even consistent there... Some SiLabs firmware uses channels > 128 for the additional pages and some use chan/page. It may be the best way to deal with it is to move the "is_valid_channel" into the device driver code so each driver can make it's own mind up.

For SiLabs I have no choice - the page and channel are combined to create the correct config byte:

#subghz channels must be calculated to include page (3 LSB page + 5 LSB channel)
# e.g. page 28 channel 15 == '100' + '01111' = '10001111' = 0x8F

Reply:
I see. Thanks for clarification.
Actually I was planning to add the channels in the device driver code, but since it was only necessary in this tool, I added it there.